great-idea/index.html

```diff
@@ -13,11 +14,88 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+<div class="container">
 
   <body>
+    <nav>
+      <div class="menu">
+        <a href='services.html'>Services</a>
+        <a href='#'>Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </div>
+      <div class="menu-logo">
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </div>
+    </nav>
+    <header class="intro-home">
+      <div class="intro">
+        <h2>Innovation</h2>
+        <h2>On</h2>
+        <h2>Demand</h2>
```

Comment:
I feel this could be be an `<h1>` for better semantic HTML. 
You could also use `<br>` HTML tags to break to a new line without having to use multiple headings. Check them out!